gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/ExplodeBundle.java

```diff
@@ -45,10 +56,14 @@ void explodeBundle(String folderPath, FilterConfiguration filterConfiguration, F
         final BundleBuilder bundleBuilder = ExportPluginModule.getInjector().getInstance(BundleBuilder.class);
         Bundle bundle = bundleBuilder.buildBundle(bundleDocument.getDocumentElement());
 
+        //checks if bundle has specified folderpath
+        if (!bundleContainsFolderPath(bundle, folderPath)) {
+            throw new BundleLoadException("Unable to find " + folderPath + " in the Gateway from the specified Gateway connection folder path");
```

Comment:
simple improvement in the message here, I think it's better something like this:
`Specified folder + folderPath + does not exist in the target gateway.`